src/components/tasks/card/AssigneeDropdownMenu.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC, useState, useEffect, useMemo } from "react";
+
+import userType from '@/interfaces/user.type';
+import { ALL_USERS } from '@/components/constants/url';
+import fetch from '@/helperFunctions/fetch';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+
+import classNames from '@/components/tasks/card/card.module.scss';
+
+type Props = {
+  updateAssignee: (username: string) => void
+}
+
+//https://github.com/Real-Dev-Squad/website-status/issues/303
+
+const AssigneeDropdownMenu: FC<Props> = ({ updateAssignee }) => {
+  const [searchValue, setSearchValue] = useState("");
+  const [filteredList, setFilteredList] = useState([]);
+  const [assigneeList, setAssigneeList] = useState([])
+  const { ERROR } = ToastTypes;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const { requestPromise } = fetch({ url: ALL_USERS });
+        const { data } = await requestPromise;
+        const usersData = data.members || data.users
+        setAssigneeList(usersData)
+      } catch (err: any) {
+        toast(ERROR, err.message);
+      }
+    };
+    fetchData();
+
+  }, []);
+
+  useEffect(() => {
+    if (searchValue) {
+      const newFilteredList = assigneeList.filter((member: userType) => (
+        member.username?.toLowerCase().includes(searchValue.toLowerCase())
+      ));
```

Comment:
nit: is it possible to check in alphabetical order, like for an 'a' we could just get people whose names start with a such as Alish, Ankush,... We could utilize binary search to reduce complexity as well.

**This is a Nit comment**

Reply:
Is this comment still unresolved ? @Muralidhar22

Reply:
yes we can close this, if optimization required, we can raise a new PR